types/record.go

```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 
 	"github.com/alanctgardner/gogen-avro/generator"
-	mapstruct "github.com/rikonor/go-mapstruct"
+	mapstruct "github.com/securityscorecard/go-mapstruct"
```

Comment:
I believe this needs to be `"github.com/securityscorecard/gogen-avro/mapstruct"`